### PR TITLE
Use consistent message for all conditions

### PR DESCRIPTION
**Pull request by kajinamit**
This updates condition messages so we use the same format for all conditions.

### CODE REVIEW

File: modules/common/condition/conditions.go [hunk 1]
```diff
@@ -219,10 +219,10 @@ const (
 	// NetworkAttachmentsReady condition messages
 	//
 	// NetworkAttachmentsReadyInitMessage
-	NetworkAttachmentsReadyInitMessage = "NetworkAttachments not started"
+	NetworkAttachmentsReadyInitMessage = "NetworkAttachments checked"
```
Comment (stuggi) [inline modules/common/condition/conditions.go:222]: `NetworkAttachments not checked` ?

File: modules/common/condition/conditions.go [hunk 2]
```diff
@@ -219,10 +219,10 @@ const (
...
 
 	// NetworkAttachmentsReadyMessage
-	NetworkAttachmentsReadyMessage = "NetworkAttachments completed"
+	NetworkAttachmentsReadyMessage = "NetworkAttachments not checked"
```
Comment (stuggi) [inline modules/common/condition/conditions.go:225]: `NetworkAttachments checked` ?

**Review by stuggi — COMMENTED** [review]
this will most likely required updates to kuttl assert s ?

File: modules/common/condition/conditions.go (hunk 1 above)
Comment (kajinamit) [reply modules/common/condition/conditions.go]: Fixed. Good catch !

File: modules/common/condition/conditions.go (hunk 2 above)
Comment (kajinamit) [reply modules/common/condition/conditions.go]: Fixed in the latest version.

**Comment by kajinamit** [comment]
> this will most likely required updates to kuttl assert s ?

A few operators ( placement-operator ) assert condition messages so we have update these, but I generally prefer not asserting these messages in kuttl, because strictly asserting the messages are not very much useful while it requires more frequent update caused by underlying change.

**Review by stuggi — APPROVED** [review]
/lgtm

**Comment by stuggi** [comment]
> > this will most likely required updates to kuttl assert s ?
> 
> A few operators ( placement-operator ) assert condition messages so we have update these, but I generally prefer not asserting these messages in kuttl, because strictly asserting the messages are not very much useful while it requires more frequent update caused by underlying change.

I think it makes sense what you said. do you plan to follow up on this?

**Comment by kajinamit** [comment]
> I think it makes sense what you said. do you plan to follow up on this?

Quickly searching the repositories it seems only mariadb-operator and placement-operator assert every single condition (including messages) so I've submitted PR to update these.

https://github.com/openstack-k8s-operators/placement-operator/pull/18
https://github.com/openstack-k8s-operators/mariadb-operator/pull/118

**Comment by kajinamit** [comment]
We later learned we have to reconsider the implementation to assert conditions (see https://github.com/openstack-k8s-operators/placement-operator/pull/18 ) so this may later require changes in kuttl tests.

**Pull request closed without merging**